### PR TITLE
RA-1777:Added  i18n support for labels on Clinical Dashboard

### PR DESCRIPTION
**Pull request by HerbertYiga**
Ticket ID: https://issues.openmrs.org/browse/RA-1777

Description
Added  i18n support for labels on Clinical Dashboard

### CODE REVIEW

**Comment by HerbertYiga** [comment]
Made a pr for this here https://github.com/openmrs/openmrs-module-coreapps/pull/332 cc @ibacher  @dkayiwa  @mks-d

**Comment by HerbertYiga** [comment]
@ibacher  hi ian for this i added languages which am sure of,ie Swahili and Spanish,for the other properties file,could i just add the english version or i can leave them empty?

**Comment by HerbertYiga** [comment]
hi @dkayiwa   what do you think on my comment above on this

**Comment by HerbertYiga** [comment]
@ibacher   hi ian,what do you think of this?

**Comment by ibacher** [comment]
@HerbertYiga Sorry for the delayed response. Our default seems to be to leave the English text in to allow someone who knows the language to come along later and provide an appropriate translation.

File: api/src/main/resources/messages_es.properties [hunk 1]
```diff
@@ -12,10 +12,10 @@
 # Copyright (C) OpenMRS, LLC.  All Rights Reserved.
 #
 
-coreapps.title=Módulo de aplicaciones centrales
-coreapps.patientDashboard.description=Aplicación Registro de Paciente
+coreapps.title=M�dulo de aplicaciones centrales
+coreapps.patientDashboard.description=Aplicaci�n Registro de Paciente
```
Comment (HerbertYiga) [inline api/src/main/resources/messages_es.properties:16]: Hi @ibacher thanks for the response on this,i am closing it so as i make a better pr without all these unwanted code formats

**Pull request closed without merging**